pom.xml

```diff
@@ -306,6 +315,62 @@
                     </execution>
                 </executions>
             </plugin>
+            <!--
+            <plugin>
+                <groupId>com.exasol</groupId>
+                <artifactId>error-code-crawler-maven-plugin</artifactId>
+                <version>0.7.1</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>verify</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            -->
```

Comment:
For now I commented this plugin. 
It throws the errors below:
```
[ERROR] E-ECM-31: Invalid incomplete builder call at ErrorMessageBuilderTest.java:19.                                                                                                           
This typically happens when you assign the ErrorMessageBuilder to a local variable and then call `toString()` on that variable. Doing so is not allowed since it makes it impossible to determin
e all components of the error declaration using static code analysis. (You could for example use if statements to select a message). Declare the error message in on fluent-programming call. 
```
I think it should be okay to have it like that in tests.

Reply:
please open a ticket at error-code-crawler. Assigning the message to a variable should be allowed. Only the builder must not be assigned.

Reply:
The you can add a comment like: Dsiabled bacause of issue-link.